common/meta/api/src/schema_api_test_suite.rs

```diff
@@ -2465,7 +2596,16 @@ impl SchemaApiTestSuite {
                 },
             };
 
+            let get_db_req = GetDatabaseReq {
+                inner: DatabaseNameIdent {
+                    tenant: tenant.to_string(),
+                    db_name: db_name.to_string(),
+                },
+            };
```

Comment:
Consider creating a function for building a `GetDatabaseReq`, to reduce lines of codes.

Something like `fn reg_get_db(tenant: impl ToString, db_name: impl ToString) {...}`

Reply:
How about adding a new function or macro like:

```rust
ASSERT_SEQ_INCR(mt.rename_table(..), mt, tenant1, db1, ...);
```

Reply:
> How about adding a new function or macro like:
> 
> ```rust
> ASSERT_SEQ_INCR(mt.rename_table(..), tenant1, db1, ...);
> ```

It will encapsulate too many things in one function and make it unclear what it tries to assert.